REVIEW.md

```diff
@@ -0,0 +1,219 @@
+# Reviewer Guidelines
+
+Reviewing pull requests (PRs) can take many forms and be reflected across many styles.
+To help make PR review more clear for contributors, the following guidelines are designed to scope review and specify what areas of feedback block PRs from merging.
+
+## Contents
+
+[Feedback](#feedback)
+
+[Expected reviewer behavior](#expected-reviewer-behavior)
+
+[For code-focused review](#for-code-focused-review])
+- [Required review](#required-review)
+- [Optional review](#optional-review)
+
+[For documentation-focused review](#for-documentation-focused-review)
+- [Required review](#required-review-1)
+- [Optional review](#optional-review-1)
+- [Followup-only items](#followup-only-items)
+- [Rebasing and merging](#rebasing-and-merging)
+
+[Communication Guidelines](#communication-guidelines)
+
+[References](#references)
```

Comment:
I'd strongly advise against (apparently) manually creating and updating a hardcoded table of contents like this. No other document in the Spyder repo has one, even those longer and more complex than this one, whereas GitHub already generates a one automatically (albeit one that could be a bit more discoverable, though we could simply call it out in the text) that can be browsed and searched by clicking the icon in the top-left corner of the document.

Generating and maintaining one without automated tooling support, instead of simply using the existing header structure, is a nightmare to maintain, as not only is it easy to drift out of sync with the actual document if any headings are added, removed or re-organized, but the links will also invisibly break (in addition to just being out of date) if any of the headers are renamed, or even (in some cases) are re-ordered. 

The simplest solution for now is just relying on the GitHub one, as we do for all other such Markdown files here, and if more visibility is needed, we can just explicitly call it out in the text in lieu of duplicating the whole thing. However, if for whatever reason we do need one in the text itself, there are a number of better ways to achieve this; for the docs, website and other repos, we use DocToC via Pre-Commit, which automatically generates and updates the ToCs automatically as needed whenever users commit new changes to the file.

Reply:
I've removed this section per request.

Some things I'd like to note anyways:
- I'm not actually sure what autogenerated GitHub table of contents you mean, so I'm just trusting you here.
- I think the fact that other Spyder documents lack this is not a reason to continue creating difficult-to-navigate documents.
- This doesn't matter to me enough to block the PR on keeping the table of contents. Iteration is welcome.

Reply:
As mentioned, its not as obvious as it could be, if you want we can explicitly call it out in the text, but here's what I was referring to:

![image](https://user-images.githubusercontent.com/17051931/143012237-daee7438-0bbb-4d34-9c4d-65f022858e59.png)

> I think the fact that other Spyder documents lack this is not a reason to continue creating difficult-to-navigate documents.

I certainly don't think this should be so either, and in fact on all the Spyder repos that I'm the primary maintainer of (website, docs, theme, etc) we have a pre-commit hook that autogenerates and updates an explicit in-text table of contents on all documents longer than a page or two, but I do think that makes it out of scope for this PR and more appropriate for a dedicated one that it applies it to all, if needed. As a much simpler solution for now, we can call specific attention to the GitHub-generated ToC in the text instead of manually duplicating it.